src/internal.c

```diff
@@ -658,6 +677,8 @@ static int ExportKeyState(WOLFSSL* ssl, byte* exp, word32 len, byte ver,
     c32toa(keys->sequence_number_hi, exp + idx);      idx += OPAQUE32_LEN;
     c32toa(keys->sequence_number_lo, exp + idx);      idx += OPAQUE32_LEN;
 
+    #if defined(WOLFSSL_DTLS)
+    if (!isTLS) {
```

Comment:
Should this be written as `isTLS != WOLFSSL_EXPORT_TLS`? I see `WOLFSSL_EXPORT_DTLS = 0` and `WOLFSSL_EXPORT_TLS = 1` being used.